POC/ChineseGPT4Vision.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    api_key = "" #attach own API key, make sure OpenAI is installed.
</pre><p>Better to read from environment variable so user doesn't have to change the code, but this is better than hardcoding the key and possibly leaking it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>